.github/pull_request_template.md

```diff
@@ -16,7 +16,7 @@ Tell reviewers how to assess your changes.
 
 ## Changelog
 
-Does the change significant enought to warrant a mention in the changelog?
+Is the change significant enought to warrant a mention in the changelog?
```

Comment:
Maybe "If this change is significant (for example, launching a new feature or deprecating a feature), you should update the changelog found on the index.html.md.erb page under the heading 'Documentation updates'."

Reply:
that reads better, will use that copy and push out the change